### DISCUSSION TITLE
creating a channel on Telegram

### DISCUSSION THREAD

**Discussion by dsanttos12** [ClassicOldSong/Apollo#72] [General]
It would be a great idea for you to create a Telegram channel to support Apollo and Artemis users. This channel would be excellent for anyone who wants to contribute to their projects.

**Comment by ClassicOldSong** [comment]
As said in the README, no real time chat support will ever be provided.

This is for preventing things like what happened in the Moonlight and Sunshine's discord happening again out of control.
